swupd-add-pkg

```diff
@@ -71,15 +71,40 @@ if [[ -n "$pkg" && -n "$bundle" ]]; then
 # [MAINTAINER]: Mixer User" | sudo tee "$BUNDLE_DIR/$bundle" > /dev/null
 	fi
 
-	# Write package to bundle definition
-	echo "$pkg" | sudo tee -a "$BUNDLE_DIR/$bundle" > /dev/null
+	grep "$pkg" "$BUNDLE_DIR/$bundle"
+	if [ $? -ne 0 ]; then
+		# Write package to bundle definition
+		echo "$pkg" | sudo tee -a "$BUNDLE_DIR/$bundle" > /dev/null
+	fi
+fi
+
+# Save LAST_VER so we can add unchanged bundles to new Manifest.MoM
+PREVIOUS_VER=$(cat "$MIX_DIR/update/image/LAST_VER")
+if [ $? -ne 0 ]; then
+	echo "LAST_VER not found, setting to 0"
+	PREVIOUS_VER=0
 fi
 
+mixver=$(cat $MIX_DIR/mixversion)
+
 # Copy the rpms into the local database location
 sudo mixer add-rpms --config "$BUILDCONF"
 sudo mixer bundle add "$bundle" os-core --config "$BUILDCONF"
-sudo mixer build chroots --config "$BUILDCONF" --new-chroots
-sudo mixer build update --config "$BUILDCONF" --new-swupd --increment
+# Use the original (non-merged) MoM for creating updates between custom content
+if [ $PREVIOUS_VER -eq 0 ]; then
+	sudo mixer build chroots --config "$BUILDCONF" --new-chroots
```

Comment:
`mixer build bundles` or are we just going with the alias until docs are updated?